Packages.props

```
@@ -24,6 +24,7 @@
     <PackageReference Update="Microsoft.VSSDK.BuildTools" Version="17.3.2078" />
     <PackageReference Update="MSBuild.ProjectCreation" Version="8.0.0" />
     <PackageReference Update="Shouldly" Version="4.0.3" />
+    <PackageReference Update="System.Configuration.ConfigurationManager" Version="5.0.0.0" />
```

Shouldn't it be `6.0.0`?

```suggestion
    <PackageReference Update="System.Configuration.ConfigurationManager" Version="6.0.0.0" />
```

---

Ah. You already merge it. Never mind.

---

It doesn't _really_ matter since I just added it explicitly so I could tell the build system it doesn't need to end up in the output directory.  But of course now dependabot is trying to update it 😄 

- [ ] https://github.com/microsoft/slngen/pull/397